### PR TITLE
resolve issues with connection timeouts etc

### PR DESCRIPTION
**Pull request by activemq**
- [x] Fixed issues with browser-stack time outs

- [x] better reporting in the dashboard

- [x] fix issues with some methods not working when running in browserstack mode

- [x] clean up yml files

### CODE REVIEW

File: test/admin-hpa/Gemfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ gem 'rb-readline'
 gem 'pry'
 gem 'require_all'
 gem 'rspec'
-gem 'selenium-webdriver'
+gem 'selenium-webdriver', '4.0.0.alpha1'
```
Comment (jon-shipley) [inline test/admin-hpa/Gemfile:13]: Alpha?

Comment (activemq) [reply test/admin-hpa/Gemfile:13]: @jon-shipley -  its the only version that works with it atm.

Comment (jon-shipley) [reply test/admin-hpa/Gemfile:13]: Okay.  Let's keep an eye on it then and upgrade when it's stable 😓

Comment (activemq) [reply test/admin-hpa/Gemfile:13]: @jon-shipley - For sure, using this version at least gets the tests up and running. Agree to keep an eye on it

**Pull request merged**